### PR TITLE
Changelog 55

### PR DESCRIPTION
**Pull request by mfeifer**
First changelog for this repository.
6 PRs

[skip ci]

### CODE REVIEW

**Comment by mfeifer** [comment]
@gmcculloug can you review?  and created documentation label?
(or anyone with merge rights on this repo)

**Comment by mfeifer** [comment]
@miq-bot add_label documentation

**Comment by gmcculloug** [comment]
@tinaafitz @mkanoor Please review.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+### Changed
+- Generic Service State Machine method update. [(#51)](https://github.com/ManageIQ/manageiq-content/pull/51)
+- Generic Service State Machine - modified update_status Automate test. [(#52)](https://github.com/ManageIQ/manageiq-content/pull/52)
+
+
+### Fixed
+
+- Generic State Machine provision instance fix [(#54)](https://github.com/ManageIQ/manageiq-content/pull/54)
+- Add openstack cloud tenant events [(#59)](https://github.com/ManageIQ/manageiq-content/pull/59)
```
Comment (gmcculloug) [inline CHANGELOG.md]: @mfeifer This should moved under `Added`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Unreleased - as of Sprint 55 end 2017-02-27
+
+### Added
+- Automate method to list ansible credentials [(#53)](https://github.com/ManageIQ/manageiq-content/pull/53)
+- Generic Service State Machine methods modified to use Service object. [(#58)](https://github.com/ManageIQ/manageiq-content/pull/58)
+
+### Changed
+- Generic Service State Machine method update. [(#51)](https://github.com/ManageIQ/manageiq-content/pull/51)
+- Generic Service State Machine - modified update_status Automate test. [(#52)](https://github.com/ManageIQ/manageiq-content/pull/52)
```
Comment (gmcculloug) [inline CHANGELOG.md]: This is just test refactoring, no need to list it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+## Unreleased - as of Sprint 55 end 2017-02-27
+
+### Added
+- Automate method to list ansible credentials [(#53)](https://github.com/ManageIQ/manageiq-content/pull/53)
+- Generic Service State Machine methods modified to use Service object. [(#58)](https://github.com/ManageIQ/manageiq-content/pull/58)
```
Comment (gmcculloug) [inline CHANGELOG.md]: This should move under `Changed`.

**Comment by mfeifer** [comment]
@gmcculloug I think I capture all of your changes.

**Pull request merged**